client/views/room/Header/Header.js

```diff
@@ -76,6 +78,9 @@ const RoomHeader = ({ room, topic }) => {
 				<Favorite room={room} />
 				<Encrypted room={room} />
 				<Translate room={room} />
+				{ t === 'l' && <Box mis='x20' display='flex'>
```

Comment:
I'm missing an image here to check the visual, not sure if the actions should not be the same like 'Favorite/Encrypted/Translate'

Reply:
Please check to description